client/app/services/api/series.js

```diff
@@ -1,5 +1,7 @@
 import axios from 'axios';
 
 const getSeries = () => axios.get('./series.json');
+const getSerieById = () => axios.get('../serie.json');
+const searchSeriesByTitle = () => axios.get('./search.json');
 
-export { getSeries };
\ No newline at end of file
+export { getSeries, searchSeriesByTitle, getSerieById };
```

Comment:
I hope it is only a mock with the purpose os adding and using axios. No needed if you only try to read jsons.

Reply:
Yes sir.